### PR TITLE
Support for `perspective` CSS property

### PR DESCRIPTION
**Pull request by rcj-siteimprove**
CSS property `perspective` has been added.

### CODE REVIEW

**Comment by rcj-siteimprove** [comment]
!pr extract

**Comment by rcj-siteimprove** [comment]
!pr extract

File: packages/alfa-css/src/value/transform/perspective.ts [hunk 1]
```diff
@@ -75,19 +75,23 @@ export namespace Perspective {
     return value instanceof Perspective;
   }
 
+  const parseLength = filter(
```
Comment (Jym77) [inline packages/alfa-css/src/value/transform/perspective.ts:78]:
```suggestion
  const parseDepth = filter(
```

File: packages/alfa-css/src/value/transform/perspective.ts [hunk 2]
```diff
@@ -75,19 +75,23 @@ export namespace Perspective {
...
-    CSSFunction.parse(
-      "perspective",
-      filter(
-        Length.parse,
-        // {@link https://drafts.csswg.org/css-values/#calc-range}
-        (length) => length.hasCalculation() || length.value >= 0,
-        () => "Depth cannot be less than 0",
-      ),
-    ),
+    CSSFunction.parse("perspective", parseLength),
```
Comment (Jym77) [inline packages/alfa-css/src/value/transform/perspective.ts:89]:
```suggestion
    CSSFunction.parse("perspective", parseDepth),
```

File: packages/alfa-css/src/value/transform/perspective.ts [hunk 3]
```diff
@@ -75,19 +75,23 @@ export namespace Perspective {
...
     ([_, depth]) => Perspective.of(depth),
   );
+
+  /**
+   * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/perspective}
+   */
+  export const parseProp = map(parseLength, (depth) => Perspective.of(depth));
```
Comment (Jym77) [inline packages/alfa-css/src/value/transform/perspective.ts:96]:
```suggestion
  export const parseProp = map(parseDepth, Perspective.of);
```

**Review by Jym77 — CHANGES_REQUESTED** [review]
Renaming an abstraction.

**Review by Jym77 — APPROVED** [review]

**Pull request merged**